Small setup improvements

swtyree opened the pull request:
Hi @ylabbe and @manuelli,

I've been using MegaPose and have been impressed with how you streamlined it since I last used it! Here is a PR with a few changes to simplify the setup process:
- I removed the requirement to set environment variables (unless following the Docker installation) by making `local_data` the default data directory and moving the rclone downloads to the seemingly defunct `src/megapose/scripts/downloads.py` script.
- I updated the readme to use the downloads script instead of using rclone directly.
- I removed `import eigenpy` and `eigenpy.switchToNumpyArray()` from `src/megapose/lib3d/transform.py` since the current version of `eigenpy` (via `conda install eigenpy -c conda-forge`) doesn't have that function and `eigenpy` is unused otherwise.

I hope these are helpful changes! Feel free to ping me with any issues.

--Stephen

ylabbe commented:
Hi @swtyree, thanks for the changes; the download script and eigenpy fixes are helpful.

I just changed the hosting location from google drive to an Inria server. Could you rebase on master and update the download script accordingly ? I believe it should mostly be about changing RCLONE_ROOT in  [src/megapose/scripts/download.py](https://github.com/megapose6d/megapose6d/pull/34/files#diff-6aa40424de59845cd615439f85f374c8593b77cc4c83452a64afd9029284f5d1)

swtyree commented:
Hi Yann, thanks for fixing the server issue! I've merged that change into my PR branch.

I also added a small fix for logging, which wasn't displaying in my terminal. According to https://stackoverflow.com/a/56144390, "depending on your runtime environment and logging levels" logging entries might not show up without first calling `logging.basicConfig()`. So I added that line to `src/megapose/utils/logging.py`, and logging now works on my system.

Let me know if you need anything else before merging!

ylabbe commented:
looks good to me, thank you!
